Fixed the seek function

WaseemALTamer opened the pull request:
Fixed the seek function so it is precise so it dose not round it to the nearest 5 seconds i also added a var for speeding or slowing down the footage

WaseemALTamer committed:
Add files via upload

WaseemALTamer committed:
Add files via upload

PaulleDemon commented:
Hey thanks for the pull request, I haven't tested or fully reviewed it yet as I am busy. 

However, couple of quick suggestion, remove `__pycache__` file add it to `.gitignore`.

make sure you are following the PEP8 style guidelines, including the naming convention. 

I'll free up myself in a month or two, and review the code properly before merging. Till then thank you for your patience and understanding

WaseemALTamer committed:
Delete tkVideoPlayer/__pycache__ directory

WaseemALTamer committed:
Add files via upload

WaseemALTamer committed:
Merge branch 'PaulleDemon:master' into master

WaseemALTamer committed:
Add files via upload

fixed the time between each frame as it was using sleep.time() which gave me trouble as the videos i was playing was not in the correct frame rate i used a simple module to correct the frame time which takes count of the time code take to run.

i also put the module into the project it is a simple module i wrote
